common/include/serial_port.h

```diff
@@ -13,28 +15,27 @@ using namespace boost::asio;
 
 class SerialPort: public AbstractDevice {
 public:
-	static const constexpr char* kPortName = "port_name";
-	static const constexpr char* kData = "data";
-	static const constexpr char* kBaudRate = "baud_rate";
+	static const constexpr char* kPortName 		= "port_name";
```

Comment:
No need to force the equal mark to be aligned, adding a space between them should be good enough.